mtags/src/main/scala-2/scala/meta/internal/pc/MetalsGlobal.scala

```diff
@@ -194,80 +195,126 @@ class MetalsGlobal(
       isVisited += key
       val result = tpe match {
         case TypeRef(pre, sym, args) =>
-          val ownerSymbol = pre.termSymbol
```

Comment:
Changes in this class come from https://github.com/scalacenter/scalafix/blob/69c0369ab49c41b0473ca68fe0a8ade396d3157e/scalafix-rules/src/main/scala/scala/meta/internal/pc/ScalafixGlobal.scala#L166